src/client/shadows/dynamicshadowsrender.cpp

```diff
@@ -297,10 +366,10 @@ video::ITexture *ShadowRenderer::getSMTexture(const std::string &shadow_map_name
 }
 
 void ShadowRenderer::renderShadowMap(video::ITexture *target,
-		DirectionalLight &light, scene::E_SCENE_NODE_RENDER_PASS pass)
+		DirectionalLight &light, int section, int total_sections, scene::E_SCENE_NODE_RENDER_PASS pass)
```

Comment:
There is no point is passing these here, the variables are already available to the method